src/core/state/rainbowChainAssets/index.ts

```diff
@@ -135,5 +137,5 @@ export const useRainbowChainAssetsStore =
         }
         return state;
       },
-    },
+    }),
   );
```

Comment:
This might exist outside of the PR as well, but noticed an issue where if we add a custom network (in this case Ronin via the dapp flow with Chainlist), the custom network asset becomes pinned and all of the other assets are filtered out. <img width="451" height="663" alt="Image" src="https://github.com/user-attachments/assets/131d4ede-d8bf-4314-aa03-8e085405b49d" />